ang/volunteer/Project.js

```diff
@@ -75,6 +73,8 @@
 
     var relationships = {};
     if(project.id == 0) {
+      project = supporting_data.values.defaults;
```

Comment:
Overwriting the existing object seems bad, even though at this point there isn't much in it. How about instead:

```javascript

project = _.extend(supporting_data.values.defaults, project);
```

This way, we use the defaults as the baseline object and anything that has already been defined in the project object (e.g., the ID) overrides the defaults. This has no effect now, but may prevent future head-scratchers as the code base matures.